From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Calculinux <calculinux@users.noreply.github.com>
Date: Sat, 29 Aug 2026 00:00:00 +0000
Subject: rockchip: rk3506: Default stdout to serial,vidconsole

Keep serial active and also send console output to the LCD vidconsole
when DM_VIDEO is enabled.

Signed-off-by: Cursor Agent <cursor@calculinux>
---
--- a/include/configs/rk3506_common.h
+++ b/include/configs/rk3506_common.h
@@ -14,7 +14,10 @@
 #define SDRAM_MAX_SIZE			0xc0000000
 
 #ifndef ROCKCHIP_DEVICE_SETTINGS
-#define ROCKCHIP_DEVICE_SETTINGS
+#define ROCKCHIP_DEVICE_SETTINGS \
+	"stdin=serial\0" \
+	"stdout=serial,vidconsole\0" \
+	"stderr=serial,vidconsole\0"
 #endif
 
 #define ENV_MEM_LAYOUT_SETTINGS			\
